cmd/frontend/graphqlbackend/git_commit.go

```diff
@@ -156,10 +166,32 @@ func (r *GitCommitResolver) Subject(ctx context.Context) (string, error) {
 	if err != nil {
 		return "", err
 	}
+
+	// Special handling for perforce depots converted to git using p4-fusion. We want to return the
+	// subject from the original changelist and not the subject that is generated during the
+	// conversion.
+	//
+	// For depots converted with git-p4, this special handling is NOT required.
+	if r.repoResolver.IsPerforceDepot() && strings.HasPrefix(commit.Message.Body(), "[p4-fusion") {
+		subject, err := parseP4FusionCommitSubject(commit.Message.Subject())
+		if err == nil {
+			return subject, nil
+		} else {
+			// If parsing this commit message fails for some reason, log the reason and fall-through
+			// to return the the original git-commit's subject instead of a hard failure or an empty
+			// subject.
+			r.logger.Error(err.Error())
```

Comment:
use our error type rather than this message. I think generally we like static strings as the logger message.
```suggestion
			r.logger.Error("failed to parse p4 fusion commit subject", log.Error(err))
```